content/reference/routing.md

```diff
@@ -0,0 +1,210 @@
+# Routing
+Choo is built up out of two parts: stores and views. In order to render a view,
+it must be added to the application through `app.route()`. This is the router.
+
+In many other frameworks, routing is done by a separate library. We found that
+routing is common in most applications, so making it part of the framework
+makes sense.
+
+Routing has a few parts to it. Routing must handle the browser's history API
+(e.g.  going forward & backwards). It must handle anchor tags, and programmatic
+actions. There's more than a few moving pieces.
+
+To perform routing, Choo uses a [Trie](https://en.wikipedia.org/wiki/Trie) data
+structure. This means that our routings is fast, and the order in which routes
+are added doesn't matter.
+
+_Note: It's recommended to read the [views]('/reference/views) chapter first, as
```

Comment:
is the `'/` intentional?

Reply:
Yeah, it was — given how the site will probably be constructed, we should link from root. Probably needs to double check once we have more of a site tho, haha